tests/integration/test_charm.py

```diff
@@ -187,6 +188,45 @@ async def test_exporter_endpoints(ops_test: OpsTest):
     assert jmx_resp.ok
 
 
+@pytest.mark.abort_on_fail
+@pytest.mark.log_level_change
```

Comment:
Leftover from local testing?

Reply:
It was a copy and paste from Zookeeper, where we used these marking, see [here](https://github.com/canonical/zookeeper-operator/blob/main/tests/integration/test_password_rotation.py#L28). 

It does not seem we do this for Kafka, and there is no value of adding this here either. Removing it. We can introduce this marking to a later stage if needed.